Highest power of 2.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+Input: N = 9
+Output: 8
+Explanation:
+Highest power of 2 less than 9 is 8.
+
+
+
+Input: N = -20
+Output: -32
+Explanation:
+Highest power of 2 less than -20 is -32.
+
+Input: N = -84
+Output: -128
+*/

```

Comment:
User input a number and we reach highest power of 2 which is close to that number